phpunit.xml.dist

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you let me know why did you rename it?

Reply:
I think phpunit's best plactice.
But I think fix is unnecessary.
So I will restore it.